docs/release-notes/release-notes-0.8.0.md

```diff
@@ -0,0 +1,109 @@
+
+Note that `./install.sh -?` will show the options for installation.
+
+Notable changes
+===============
+
+### Native segwit (bech32) order book
+
+See the [native segwit upgrade guide](../NATIVE-SEGWIT-UPGRADE.md) for details. From this release, new wallets will be created as native segwit by default. This setting can be changed by setting `native=false` in the `[POLICY]` section of `joinmarket.cfg`.
+
+*Note that native segwit wallets are not a new feature: we have had them since [version 0.5.1](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.5.1.md#add-native-segwit-wallet-to-backend) and they have been used for payjoin experimentation*, what is new is only the default, and the use in Joinmarket coinjoins.
+
+If you create wallets, and run Takers and Makers with the default settings, you will be using only native segwit for your coinjoins and other transactions, and will only join with other participants using native segwit. Note that everything about pre-existing p2sh coinjoins and transactions still works as before, if you disable as per the above `native=false`.
+
+`ab87db2` Create native segwit v0 order type
+
+`2ae348b` Update signature exchange and verification for bech32
+
+`3c96d41` Several tweaks for bech32 orderbook, plus docs
+
+`ebd54d6` Bugfix: yg-privacyenhanced non-integer fees
+
+`3537b70` Update transaction parameters
+
+`9e66bc5` Update ob-watcher
+
+#### Bugfix: correct minsize calculation for yield generators with reloffers, and txfee contribution
+
+A bug in the calculation of yield generator minsize, for reloffers, was previously not resetting it to a higher value to account for the txfee contribution; this is fixed. A consequence of this: default txfee contribution value is changed from 100 to 0, to avoid confusing yield generator users who might not understand the reason their number of coins is insufficient (it is a function of both the relative fee setting and the txfee contribution setting, to ensure that the profit is always at least 20% of the latter). The complexity of this should probably be removed in future updates to the protocol.
+
+`dcfc610` Bugfix: allow minsize dynamic update
+
+`d1ae07d` set default txfee contrib to zero, improve error message
+
+#### Don't fail on whitespace in recovery seed entry
+
+In entering recovery seed phrases on Qt and command line, we now ensure that white space is always stripped appropriately; prior to this fix, unexpected failures could occur due to whitespace being interpreted as a word in the seed phrase.
+
+`29b274e` Strip whitespace from beginning and end of mnemonic on wallet recovery
+
+`5f07872` allow multiple whitespace in mnemonic entry
+
+#### Make libsecp256k1 tests optional in installation script
+
+This option is enabled with `--disable-secp-check`, and is useful for people on more constrained devices who want to speed up installation.
```

Comment:
```suggestion
This option is enabled with `--disable-secp-check` parameter of `install.sh` script, and is useful for people on more constrained devices who want to speed up installation.
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/734/commits/8a885aaa999225cfba1aa9314ca5bb308fb2c4fe